### ISSUE TITLE
Strings containing diacritic characters are sorted to last

### ISSUE THREAD

**Issue by alan12345678** [gnumdk/lollypop#528]
Firstly it was that "Édith Piaf" was sorted to the very bottom of the artist list (after "Tom Zé" for example).

Now I notice that, for example," Röyksopp" appear after "Rumer"

I've browsed the library in Nautilus, which sorts them correctly (strictly alphabetic), so conclude it must be the player sorting them this way.

**Comment by d9h02f** [comment]
The search also only finds results that match the diacritic marks exactly, searching for "cinematique" won't find anything called "cinématique". This could be solved by using tracker.

**Comment by alan12345678** [comment]
my issue has just resolved itself after an update

**Comment by gnumdk** [comment]
Here the issue is always here..
For performance reason, I use sqlite to order artists.

But on update, I use python...

**Comment by gnumdk** [comment]
http://stackoverflow.com/questions/13248601/sqlite-sort-special-characters-alphabetically

**Comment by gnumdk** [comment]
Only on Android :-(

**Comment by alan12345678** [comment]
my issue has only partly resolved itself after update. It's better but still not entirely correct. However, am simply going to wait for system updates etc. as, to be honest, I  have NO intention of start diving into code when all I wanted was just a player that works ;0))

**Comment by alan12345678** [comment]
simply in order to perhaps be helpful, I'll just reply here with a final
part of the solution I've found for sorting. I'm not sure how aware
other users might be with regard to name sorting:
 
I cured my problems completely by editing the file "extended tag"
information. This is not normally visible by default in my editor of
choice, (nor is it, for example, in Quod Libet) but there are such tags
as "artist sort order" and "album artist sort order", and very often
these are written during a lookup e.g. on freedb.org and often the user
(me in this case) is not invited to change these tags and they are
usually hidden within a standard tag editing window.
 
Thus my problem regarding "Cesária Évora" was cured simply by editing
and removing both of these 'hidden' tags. It now sorts itself correctly
into either "C" or "E" depending on how I wish to further edit my tags.
 
:)
 
*alan ansell figurative painting Itzac France*
He would be a poet who could impress the winds and streams into his
service, to speak for him.
- Henry David Thoreau
 
 
----- Original message -----
